src/components/il-tabs/il-tabs.component.js

```diff
@@ -94,6 +117,7 @@ class TabComponent extends LitElement {
   }
 
   render() {
+    this.setTabs();
     return html`
         <div id="container" ?compact=${this.compact}>
```

Comment:
The compact attribute is just for the polyfill right? I think it should be a CSS class